_config.yml

```diff
@@ -1,49 +1,11 @@
 project: cockroachdb
-audience: all
-product: all
-platform: all
-version: all
-destination: ../cockroach_site
 topnav_title: CockroachDB
 homepage_title: CockroachDB Docs
 site_title: Cockroach Labs
 google_analytics: UA-63775601-1
 
-# variables
-sidebar_tagline: 
-sidebar_version: 
-project_file_name: doc
-
-port: 4001
```

Comment:
You set this when you invoke jekyll itself. It doesn't need to be in the config file. You can also do it via docker.